releases.yaml

```diff
@@ -0,0 +1,16 @@
+sources:
+  - id: keycloak
+    productName: Keycloak
+    repo: keycloak/keycloak
+    githubReleaseFrom: 23.0.0
+    releaseNotes: release/${version.getVersionShorter()}/docs/documentation/release_notes/topics/${version.getVersion()?replace(".", "_")}.adoc
+    attributeSources:
+      - release/${version.getVersionShorter()}/docs/documentation/topics/templates/document-attributes.adoc
+      - ${version.getVersion()}/docs/documentation/topics/templates/document-attributes.adoc
+    migrationGuidePath: docs/latest/upgrading/index.html#migration-changes
+  - id: keycloak-client
+    productName: Keycloak Client Libraries
+    repo: keycloak/keycloak-client
+    githubReleaseFrom: 26.0.1
+    releaseNotes: main/docs/release-notes/${version.getVersion()?replace(".", "_")}.adoc
```

Comment:
What does this point to?

Reply:
This points to the content for the release notes, if present for a specific version it will add the contents on top of the changelog in the blog post. For example, [here are](https://github.com/keycloak/keycloak/tree/main/docs/documentation/release_notes/topics) the release notes for Keycloak itself.

I've made this configurable rather than conventional as the path in the main repo is quite deep, and we don't want that depth in the clients, or any future repos.

Reply:
I got that part, but not sure what `main/docs/release-notes/..` points to as that doesn't exist

Reply:
There are currently no releases of the client that warrant a release post. But after discussing that we need a `.0` version I will add some notes there to make it complete.

Reply:
Added release notes from https://github.com/keycloak/keycloak-client/issues/120.